features/calendar/components/CalendarView.tsx

```
@@ -1,47 +1,51 @@
+  const handleSelect = async (ymd: string) => {
+    if (ymd === today) {
+      try {
+        const { routines, totalCount } = await getRoutines();
+        Alert.alert("오늘의 루틴", `총 루틴: ${totalCount}\n첫번째: ${routines[0]?.title ?? "-"}`);
+      } catch {
+        Alert.alert("오류", "루틴을 불러오지 못했습니다.");
+      }
+    } else {
+      const { useRouter } = await import("expo-router");
+      const router = useRouter();
+      router.push({ pathname: "/retrospect", params: { date: ymd } });
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)